libdnf/hy-subject-private.c

```diff
@@ -28,11 +28,11 @@
 #include "hy-iutil.h"
 #include "hy-subject-private.h"
 
-#define PKG_NAME "([^:(/]+)"
+#define PKG_NAME "([^:(/=<> ]+)"
```

Comment:
but `=<> ` can't be anywhere in any part of NEVRA... probably I just don't get what's the point of this

Reply:
It should provide 0 possibilities for string like vim-filesystem-2:8.0.1171-1.fc26.x86=64. Now it provides several, that have to be tested by queries later on.